include/crm/common/bundles_internal.h

```diff
@@ -27,6 +27,26 @@ typedef struct {
     pcmk_resource_t *remote;    //!< Pacemaker Remote connection into container
 } pcmk__bundle_replica_t;
 
+/*!
+ * \internal
+ * \brief Check whether a resource is part of a bundle
+ *
+ * \param[in] rsc  Resource to check
+ *
+ * \return true if \p rsc is part of a bundle, otherwise false
+ */
+static inline bool
+pcmk__is_bundled(const pcmk_resource_t *rsc)
+{
+    if (rsc == NULL) {
+        return false;
+    }
+    while (rsc->parent != NULL) {
```

Comment:
I know this is copy-paste, but we could probably use `pe__const_top_resource` here.